libs/designsystem/src/lib/components/grid/grid.component.ts

```diff
@@ -26,7 +26,7 @@ class GridCard {
   // eslint-disable-next-line
   host: { '[attr.max-columns]': 'maxColumns' },
```

Comment:
Could consider refactoring to hostbinding here too, and if we use it elsewhere. Should be relatively low impact.